content/docs/user-guide/contributing/docs.md

```diff
@@ -70,6 +70,15 @@ project dependencies with Yarn:
 $ yarn
 ```
 
+> In Windows, you may need to install [Python], [Visual Studio Build Tools], and
+> [Windows SDK] beforehand for dependencies.
+
+[windows sdk]:
+  https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/
+[visual studio build tools]:
+  https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019
+[python]: https://www.python.org/downloads/
```

Comment:
Let's put this before all the code blocks? Otherwise it kind of interrupts the doc flow a bit.

Reply:
TBH maybe we should recommend a POSIX env instead and link to https://dvc.org/doc/user-guide/running-dvc-on-windows for details.

Reply:
I was able to run the node on Windows. This is not about running DVC itself on Windows, this is about running dvc.org clone on Windows. 

I agree that using WSL / POSIX is better. But in order to contribute to documents on Windows (without all this WSL stuff first), one should be able to run node.js first. Otherwise, for example, running `yarn format-all` doesn't work for contributors.

Reply:
I moved it before the node installation command.